arlas-core/src/main/java/io/arlas/server/core/impl/elastic/utils/ElasticClient.java

```diff
@@ -70,26 +82,88 @@ public class ElasticClient {
     private final ElasticsearchClient client;
     private final JacksonJsonpMapper mapper;
 
+    private static final X509ExtendedTrustManager TRUST_MANAGER = new X509ExtendedTrustManager() {
+        @Override
+        public void checkClientTrusted(X509Certificate[] x509Certificates, String s) throws CertificateException {
+
+        }
+
+        @Override
+        public void checkServerTrusted(X509Certificate[] x509Certificates, String s) throws CertificateException {
+
+        }
+
+        @Override
+        public X509Certificate[] getAcceptedIssuers() {
+            return new X509Certificate[0];
+        }
+
+        @Override
+        public void checkClientTrusted(X509Certificate[] x509Certificates, String s, Socket socket) throws CertificateException {
+
+        }
+
+        @Override
+        public void checkServerTrusted(X509Certificate[] x509Certificates, String s, Socket socket) throws CertificateException {
+
+        }
+
+        @Override
+        public void checkClientTrusted(X509Certificate[] x509Certificates, String s, SSLEngine sslEngine) throws CertificateException {
+
+        }
+
+        @Override
+        public void checkServerTrusted(X509Certificate[] x509Certificates, String s, SSLEngine sslEngine) throws CertificateException {
+
+        }
+    };
+
     public ElasticClient(ElasticConfiguration configuration) {
         // disable JVM default policies of caching positive hostname resolutions indefinitely
         // because the Elastic load balancer can change IP addresses
         java.security.Security.setProperty("networkaddress.cache.ttl", "60");
         java.security.Security.setProperty("networkaddress.cache.negative.ttl", "0");
 
+        SSLContext context = null;
+        try {
+            context = SSLContext.getInstance("TLS");
```

Comment:
What does that TLS designates?

Reply:
Transport Layer Security: more recent version of SSL